### ISSUE TITLE
Manifest IDs do not contain "@"

### ISSUE THREAD

**Issue by benwbrum** [ArchiveLabs/iiif.archivelab.org#7]
Manifests displayed within the top-level IIIF collection use `id` instead of `@id`, so they are failing validation with an obscure error about metadata URIs.

Example:
```json
  "collections":[
    {
      "@type":"sc:Manifest",
      "id":"https://iiif.archivelab.org/VE-IMG-17443/manifest.json",
      "label":""
    },
```
Change `"id":"https://iiif.archivelab.org/VE-IMG-17443/manifest.json"` to `"@id":"https://iiif.archivelab.org/VE-IMG-17443/manifest.json"` to fix.

**Comment by ahankinson** [comment]
I'm finding this with `type` -- it should be `@type`:

```js
"@context": "http://iiif.io/api/image/2/context.json",
"@id": "https://iiif.archivelab.org/iiif/olditalianmaster00pint$2/annotation",
"motivation": "sc:painting",
"resource": {
"@id": "https://iiif.archivelab.org/iiif/olditalianmaster00pint$2/full/full/0/default.jpg",
"@type": "dctypes:Image",
"format": "image/jpeg",
"height": 3369,
"service": {
"@context": "http://iiif.io/api/image/2/context.json",
"@id": "https://iiif.archivelab.org/iiif/olditalianmaster00pint$2",
"profile": "http://iiif.io/api/image/2/profiles/level2.json"
},
"width": 2246
},
"type": "oa:Annotation"

```

**Comment by ahankinson** [comment]
I think this was closed by #14
